### PR TITLE
PLT-462 Adding diagnostic info

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

**Comment by jwilander** [comment]
+1

File: mattermost.go [hunk 1]
```diff
@@ -71,6 +75,53 @@ func main() {
 	}
 }
 
+func diagnosticsJob() {
+	go func() {
+		for {
+			if utils.Cfg.PrivacySettings.EnableDiagnostic && model.BuildNumber != "_BUILD_NUMBER_" {
```
Comment (crspeller) [inline mattermost.go]: The second condition here is checking "dev mode". We had problems with strange settings like this before. Maybe just rely on having EnableDiagnostic disabled? You might get some bad diagnostics from external devs but those would be easy to filter out.  Or at least move this check to a `isNonOfficalBuild()` function or something similar.

**Comment by coreyhulen** [comment]
I've added an IsOfficalBuild func

**Comment by jwilander** [comment]
Still +1

**Comment by hmhealey** [comment]
Looks good

**Pull request merged**